### ISSUE TITLE
Automatically start tracking on android boot

### ISSUE THREAD

**Issue by geek1011** [schollz/find#168]
At the moment, I need to start the find app and press track every time I turn on my phone. Is it possible to start tracking the moment I turn on my phone?

**Comment by schollz** [comment]
Yes, you should checkout using Tasker and a 3rd party app called Room+.

**Comment by elRadix** [comment]
+1 to have this  within the current app

**Comment by ferdydek** [comment]
+1
